src/main/java/com/blackduck/integration/detect/configuration/DetectProperties.java

```diff
@@ -916,6 +916,17 @@ private DetectProperties() {
             .setGroups(DetectGroup.GRADLE, DetectGroup.GLOBAL)
             .build();
 
+    public static final BooleanProperty DETECT_GRADLE_ROOT_ONLY =
+            BooleanProperty.newBuilder("detect.gradle.root.only", false)
+                    .setInfo("Gradle Root Only Enabled", DetectPropertyFromVersion.VERSION_10_1_0)
+                    .setHelp(
+                            "If set to true, Gradle Native Inspector will evaluate only the root project's dependencies.",
+                            "This property trumps other inclusion/exclusion rules and therefore should not be combined with detect.gradle.excluded.projects, detect.gradle.excluded.project.paths, detect.gradle.included.projects or detect.gradle.included.project.paths."
+                    )
+                    .setGroups(DetectGroup.GRADLE, DetectGroup.SOURCE_SCAN)
+                    .setCategory(DetectCategory.Advanced)
+                    .build();
+
```

Comment:
@cpottsbd This is the only other piece that requires your review ^

Reply:
Slightest of mods. : ) 
"If set to true, Gradle Native Inspector will only evaluate root project dependencies.",
"This property overrides other inclusion/exclusion rules and therefore should not be combined with detect.gradle.excluded.projects, detect.gradle.excluded.project.paths, detect.gradle.included.projects, or detect.gradle.included.project.paths."